packages/components-native/src/Typography/Typography.tsx

```diff
@@ -83,6 +84,11 @@ export interface TypographyProps<T extends FontFamily>
    */
   readonly fontStyle?: T extends "base" ? BaseStyle : DisplayStyle;
 
+  /**
+   * Style of underline for the text
```

Comment:
same thing here

if Android doesn't support any of these values it will appear to offer let's call that out at the very least

Reply:
Addressed in e9900e023d046db9397ce4e7f083b93e9b47e7b3